packages/app-staking/src/md/basic.md

```diff
@@ -1,21 +1,21 @@
 # Nominators
 
-Nominators have a responsibility to elect the most trustworthy validators to secure the network.  
-To become a nominator you must stake the minimum required CENNZ tokens and vote for your preferred validators.  
+Nominators have a responsibility to elect the most trustworthy validators to secure the network.
```

Comment:
@KarishmaBothara  the trailing whitespace is what markdown uses to create a newline

Reply:
adding the trailing whitespace back.